src/main/kotlin/no/nav/syfo/brev/arbeidstaker/domain/ArbeidstakerBrevDTO.kt

```diff
@@ -16,4 +16,7 @@ data class ArbeidstakerBrevDTO(
     val videoLink: String?,
     val document: List<DocumentComponentDTO>,
     val virksomhetsnummer: String,
+    val svarType: String?,
+    val svarTekst: String?,
```

Comment:
Hva med å dra ut `svarType`, `svarTekst` og `svarTidspunkt` i en egen type? Bare et forslag. Ellers synes jeg det ser bra ut fra esyfo sitt point of view 👍